SU-HSDO/suhumsci
# SHS-5911: Add indicator of 3rd level shortcut menu

# READY FOR REVIEW

## Summary
Adds indicator of shortcut menu 3rd level

## Need Review By (Date)
11/20

## Urgency
'medium

## Steps to Test
1. In any site, go to this path: /admin/config/user-interface/shortcut/manage/default/customize
2. Add a shortcut item as a third level item
3. Now verify that if there are 3rd-level items, they have an indicator that let the user knows there's a 3rd level menu item
4. You can check an existing 3rd-level link here: https://pr1681-drxb2zpp0vmpul1acmcab6mkcz95fral.tugboatqa.com/  (Shortcuts > Site Actions > Page not found Report > Third level)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208751657725172
